keyboards/hackpad/hackpad.c

```diff
@@ -0,0 +1,13 @@
+// Copyright 2023 Nico Stuhlmueller (@ThePurox)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
```

Comment:
This is the default/core behaviour on the develop branch, so this file can be removed.

Reply:
Thank you. I've deleted it.